packages/core/js-sdk/src/__tests__/client.spec.ts

```diff
@@ -44,7 +44,7 @@ const server = setupServer(
     }
   }),
   http.get(`${baseUrl}/pubkey`, ({ request, params, cookies }) => {
-    if (request.headers.get("x-medusa-pub-key") === "test-pub-key") {
+    if (request.headers.get("x-publishable-api-key") === "test-pub-key") {
```

Comment:
Q: should we put that in a constant in the package? in case of future changes?

Reply:
Yeah we can, let me do that 👍